cpg-core/src/test/kotlin/de/fraunhofer/aisec/cpg/graph/edges/flows/DataflowTest.kt

```diff
@@ -101,4 +106,25 @@ class DataflowTest {
             assertEquals(0, node2.prevDFGEdges.size)
         }
     }
+
+    @Test
+    fun testThrow() {
+        val result = prepareThrowDFGTest()
+
+        // Let's assert that we did this correctly
+        val main = result.functions[0]
```

Comment:
I think the function is called "foo" and we should probably also retrieve it like `functions["foo"]` in case we extend the test and move things around.

Reply:
Done